packages/openneuro-components/src/search-page/SearchResult.stories.tsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react'
+import { Story, Meta } from '@storybook/react'
+import { mri } from '../mock-content/mri-search-results'
+
+import { SearchResult, SearchResultProps } from './SearchResult'
+
+export default {
+  title: 'Components/SearchResult',
+  component: SearchResult,
+} as Meta
+
+const SearchResultTemplate: Story<SearchResultProps> = ({ node, profile }) => {
+  return <SearchResult node={node} profile={profile} />
```

Comment:
Implementing FacetExample and SearchResults as render props would expose them here, which would allow us to wrap them in stateful containers in app and still use mock data in storybook.